geometry_msgs/PoseStamped local_pose
